DragonLoop/DragonLoopApp/DragonLoopApp/ViewModels/SchedulesViewModel.cs

```diff
@@ -35,42 +45,58 @@ private async Task ExecuteLoadSchedulesCommand()
                 SchedulesGrid.Children.Clear();
                 SchedulesGrid.RowDefinitions.Clear();
                 SchedulesGrid.ColumnDefinitions.Clear();
+                SchedulesGrid.RowDefinitions.Add(new RowDefinition());
```

Comment:
Is this in the right place? Is clearing it and then adding a new RowDefinition the way to go?

Reply:
+1

Reply:
Yes, the Grid Clear() was the commit that @vangoespow added as a hotfix. The new RowDefinition() was initially added after loading Route and Schedule, but I just grouped the Grid Definitions together.

Reply:
![Capture](https://user-images.githubusercontent.com/35273005/68537428-4094dd00-0331-11ea-83d5-420085634c59.PNG)

Reply:
^ currently the way it is